code/modules/mob/living/carbon/superior_animal/roach/roach_summon.dm

```diff
@@ -0,0 +1,66 @@
+// A one-use item that summon a specific type of roach and tame it.
+/* Here is the current list of choices :
+	- "kampfer" : Default Roach
+	- "unbekannt" : Bluespace Roach
+	- "elektromagnetisch" : EMP Roach
+	- "fuhrer" : Fuhrer Roach
+	- "gluhend" : Glowing Roach
+	- "jager" : Hunter Roach
+	- "kaiser" : Kaiser Roach
+	- "kraftwerk" : Nanite Roach
+	- "seuche" : Support Roach
+	- "panzer" : Tank Roach
+	- "gestrahlte" : Toxic Roach
+*/
+/obj/item/roach_summon
+	name = "bluespace roach beacon"
+	desc = "The result of a collaboration between Soteria and the Cht'mant hive. This beacon allow to summon a single tamed roach."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "uplink_access"
+	item_state = "uplink_access"
+	w_class = ITEM_SIZE_SMALL
+	price_tag = 2500
+	var/roach_type = "kampfer" // The type of roach to spawn.
+	var/mob/living/carbon/superior_animal/roach/roach // A reference to the roach this item summoned.
+
+/obj/item/roach_summon/attack_self(mob/user as mob)
+	switch(roach_type)
+		if("kampfer")
+			roach = new /mob/living/carbon/superior_animal/roach(src)
+		if("unbekannt")
```

Comment:
Should never be able to spawn this one, it has gold/silver/diamonds/plasma in it

Reply:
It is modular and future-proof, there is no way to change the type of roach it will spawn or to even get the thing. The only one that spawn is coded to spawn a panzer roach.